server/render-template/index.tsx

```diff
@@ -56,7 +56,7 @@ function renderTemplate(
 
   const template = baseTemplate.replace(
     '<div id="root"></div>',
-    `<div id="root">${renderToStaticMarkup(<Node {...templateData} />)}</div>`
+    `<div id="root">${renderToString(<Node {...templateData} />)}</div>`
```

Comment:
Any reason why was this chnaged the string method?

Reply:
This allowed the inline css be processed, you can see the deets [here](https://reactjs.org/docs/react-dom-server.html#rendertostring)